frontends/mit-open/src/pages/ListDetailsPage/ItemsListing.test.tsx

```diff
@@ -0,0 +1,318 @@
+import React from "react"
+import { faker } from "@faker-js/faker/locale/en"
+import { SortableList, SortableItem } from "ol-components"
+import LearningResourceCard from "@/page-components/LearningResourceCard/LearningResourceCard"
+import { factories, urls, makeRequest } from "api/test-utils"
+import {
+  screen,
+  expectProps,
+  renderWithProviders,
+  setMockResponse,
+  waitFor,
+  act,
+} from "../../test-utils"
+import ItemsListing from "./ItemsListing"
+import type {
+  ItemsListingProps,
+  LearningResourceListItem,
+} from "./ItemsListing"
+import { ControlledPromise } from "ol-test-utilities"
+import invariant from "tiny-invariant"
+import {
+  LIST_TYPE_LEARNING_PATH,
+  LIST_TYPE_USER_LIST,
+} from "@/common/constants"
+
+jest.mock("ol-components", () => {
+  const actual = jest.requireActual("ol-components")
+  return {
+    __esModule: true,
+    ...actual,
+    SortableList: jest.fn(actual.SortableList),
+    SortableItem: jest.fn(actual.SortableItem),
+  }
+})
+
+const spyLearningResourceCard = jest.mocked(LearningResourceCard)
+const spySortableList = jest.mocked(SortableList)
+const spySortableItem = jest.mocked(SortableItem)
+
+const learningResourcesFactory = factories.learningResources
+const userListsFactory = factories.userLists
+
+const getPaginatedRelationships = (
+  listType: string,
+  count: number,
+  parent: number,
+) => {
+  if (listType === LIST_TYPE_LEARNING_PATH) {
+    return learningResourcesFactory.learningPathRelationships({
+      count,
+      parent,
+    })
+  } else if (listType === LIST_TYPE_USER_LIST) {
+    return userListsFactory.userListRelationships({
+      count,
+      parent,
+    })
+  } else {
+    throw new Error("Invalid list type passed to getPaginatedRelationships")
+  }
+}
+
+describe("ItemsListing", () => {
```

Comment:
I was looking all through the Jest documentation to see if I could do this and I must have missed something because I didn't think you could. I should have just tried it!